src/frontend/js/events/auto-complete.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * this file contains function for autocompletion of the event form fields
+ */
+
+window.addEventListener("load", () => {
+    if (document.getElementById("id_start_date") && document.getElementById("id_start_time")) {
+        document.getElementById("id_start_date").addEventListener("change", autocompleteEndDate);
+        document.getElementById("id_start_time").addEventListener("change", autocompleteEndTime);
```

Comment:
```suggestion
        document.getElementById("id_start_date").addEventListener("focusout", autocompleteEndDate);
        document.getElementById("id_start_time").addEventListener("focusout", autocompleteEndTime);
```
I would set the value only after the input field looses focus, because otherwise the value gets set as soon as the start field has a valid input, e.g.:
- Type `01.01.2021` in the start date, then the end date will be `01.01.0002`
- Type `10:30` in the start time, the the end time will be `10:03`

So with `focusout`, I think the most obvious problems should be fixed.